deepspeed/runtime/zero/stage2.py

```diff
@@ -15,6 +15,7 @@
 from deepspeed.runtime.fp16.loss_scaler import LossScaler, DynamicLossScaler
 from deepspeed.runtime.utils import see_memory_usage, is_model_parallel_parameter
 from deepspeed.runtime.zero.config import ZERO_OPTIMIZATION_GRADIENTS
+from deepspeed.ops.adam import DeepSpeedCPUAdam
```

Comment:
I wonder if we need to control this import with a flag, since we may not install CPU-ADAM! We may just import it when cpu_offload is true. What do you think?

Reply:
Sure, we can do that.